src/import-data/ImportDataDestination.ts

```diff
@@ -187,6 +188,20 @@ export const ImportDataDestination = (props: ImportDataDestinationProps): ReactN
     ]);
   };
 
+  const protectedWorkspaces = workspaces.filter((workspace) => {
```

Comment:
In case the data are not marked protected, then these will just be workspaces, not protected workspaces?  A bit of a nit and I know naming is hard, but is there a more general name so that this variable isn't misleading in that case?

Reply:
no thats a good point, let me think on that

Reply:
how about "workspacesToImportTo"?

Reply:
Works for me!